File: js/tests/burp.test.js
const longBurp = require("../burp");

const data = [
  [3, "Burrrp"],
  [5, "Burrrrrp"],
  [9, "Burrrrrrrrrp"],
  [10, "Burrrrrrrrrrp"],
  [13, "Burrrrrrrrrrrrrp"],
  [18, "Burrrrrrrrrrrrrrrrrrp"],
  [1, "Burp"]
];

describe("longBurp", () => {
  describe("Return the string 'Burp' with num amount of Rs", () => {
    test.each(data)("longBurp(%i) = %s", (num, result) => {
      let actual = longBurp(num);

      expect(actual).toEqual(result);
    });
  });
});

File: CSharp/Tests/BurpTest.cs
using Xunit;

namespace CSharp.Tests
{
    public class BurpTest
    {
        [Theory]
        [InlineData(3, "Burrrp")]
        [InlineData(5, "Burrrrrp")]
        [InlineData(9, "Burrrrrrrrrp")]
        [InlineData(10, "Burrrrrrrrrrp")]
        [InlineData(13, "Burrrrrrrrrrrrrp")]
        [InlineData(18, "Burrrrrrrrrrrrrrrrrrp")]
        [InlineData(1, "Burp")]
        public void LongBurp_IntInputValue_ReturnTheWordBurpWithAmountOfRs(int num, string expected)
        {
            var actual = Burp.LongBurp(num);

            Assert.Equal(expected, actual);
        }
    }
}